fastlane/lanes/release.rb

```diff
@@ -151,23 +151,23 @@
     close_milestone(repository: GHHELPER_REPO, milestone: version)
 
     # Start the build
-    trigger_release_build(branch_to_build: "release/#{version}")
+    trigger_release_build(branch: "release/#{version}")
   end
 
   # Triggers a beta build on CI
   #
-  # @option [String] branch_to_build The name of the branch we want the CI to build, e.g. `release/19.3`
+  # @option [String] branch The name of the branch we want the CI to build, e.g. `release/19.3`
   #
   lane :trigger_beta_build do |options|
-    trigger_buildkite_release_build(branch: options[:branch_to_build], beta: true)
+    trigger_buildkite_release_build(branch: options[:branch], beta: true)
```

Comment:
tbh in the couple of cases I need to call this lane manually, I'm already often annoyed that I have to provide the branch name explicitly… and I've pondered a while ago if we shouldn't default it to `"release/#{ios_get_app_version}"` if we don't provide a `branch:` parameter explicitly from the CLI, to make things even easier to call?

```suggestion
    branch_to_build = options[:branch] || "release/#{ios_get_app_version}"
    trigger_buildkite_release_build(branch: branch_to_build, beta: true)
```

Reply:
Good idea. This default value is particularly appropriate for "beta" and "release" builds as `release/...` is the expected branch for them.

Also using `ios_get_app_version` would work most of the time, the only case I can think of resulting in an unexpected build would be if the branch from which we run this is not up-to-date with `trunk`, or if we run from `trunk` in the middle of a code freeze. Both are super-edge cases. Besides, an unexpected value in that way would be rejected by ASC, so no harm done 👍